scripts/releases/make-manylinux

```diff
@@ -72,9 +72,9 @@ if [ -d /gevent -a -d /opt/python ]; then
         # The downside is that we must install dependencies manually.
         # NOTE: We can't upgrade ``wheel`` because ``auditwheel`` depends on
         # it, and auditwheel is installed in one of these environments.
-        python -mpip install -U "cython >= 3.0a5" cffi greenlet setuptools
+        python -mpip install -U "cython >= 3.0a5" cffi 'greenlet >= 0.4.17' setuptools
```

Comment:
What do you mean 'yanked' exactly? I am asking because we had an issue in production, same as the one [here](https://github.com/gevent/gevent/issues/1674) and I am trying to get a better understanding of what happened.

Reply:
'yanked' is a feature on PyPI that prevents a release from being automatically discovered. It's still there but you have to ask for it exactly. The greenlet version was published, then when this issue was discovered it was yanked. When gevent 20.9 was released, it was unyanked.

Reply:
oh, that makes sense. Thank you very much kind sir :smile: